Ops/WebOperations.cs

```diff
@@ -201,21 +195,22 @@ public static void ProcessDNSInformation(string host, ref SiteEndpoint endpoint)
             endpoint.DestinationAddressType = address?.AddressFamily.ToString();
             endpoint.DestinationServer = hostInfo.HostName;
         }
-        public static bool CheckSQLDatabaseAccesibility(string connectionString)
+        public static async Task<bool> CheckSQLDatabaseAccesibilityAsync(string connectionString)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'CheckSQLDatabaseAccesibilityAsync' to match pascal case naming rules, consider using 'CheckSqlDatabaseAccesibilityAsync'.](https://app.codacy.com/app/onurakkaya/DSM.Core/pullRequest?prid=4732054)